### PR TITLE
use amigo amis

### PR DESCRIPTION
**Pull request by jharewinton**
includes switch to yaml for riff-raff & cloudformation

### CODE REVIEW

File: cloudformation/tag-manager.yaml [hunk 1]
```diff
@@ -0,0 +1,335 @@
...
+    Description: AMI id
+    Type: String
+  GuardianIP:
+    Description: Ip range for the office
+    Type: String
+    Default: 77.91.248.0/21
+  GithubTeamName:
+    Description: Github team name, used for giving ssh access to members of the team.
+    Type: String
+    Default: Editorial-Tools-SSHAccess
```
Comment (jennysivapalan) [inline cloudformation/tag-manager.yaml]: You can remove GithubTeamName as Amigo will do it for us

File: cloudformation/tag-manager.yaml [hunk 2]
```diff
@@ -0,0 +1,335 @@
...
+    Properties:
+      PolicyName: TagManagerStaticFilesWritePolicy
+      PolicyDocument:
+        Statement:
+        - Effect: Allow
+          Action: sts:AssumeRole
+          Resource: arn:aws:iam::642631414762:role/composerWriteToStaticBucket
+      Roles:
+      - !Ref 'TagManagerRole'
+  TagManagerGetTeamKeysPolicy:
```
Comment (jennysivapalan) [inline cloudformation/tag-manager.yaml]: This can go too

File: cloudformation/tag-manager.yaml [hunk 3]
```diff
@@ -0,0 +1,314 @@
...
+      PolicyName: TagManagerKinesisPolicy
+      PolicyDocument:
+        Statement:
+        - Effect: Allow
+          Action:
+          - kinesis:*
+          Resource: '*'
+      Roles:
+      - !Ref 'TagManagerRole'
+  TagManagerSQSPolicy:
```
Comment (currysoup) [inline cloudformation/tag-manager.yaml]: This policy can go I think

Comment (jharewinton) [inline cloudformation/tag-manager.yaml]: Raised as https://github.com/guardian/tagmanager/issues/279

**Pull request merged**